pkgs/by-name/zo/zotify/package.nix

```
@@ -0,0 +1,52 @@
+{
+  fetchFromGitHub,
+  python3Packages,
+  protobuf_21,
+  lib,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zotify";
+  version = "0.16.13";
+
+  src = fetchFromGitHub {
+    owner = "zotify-dev";
+    repo = "zotify";
+    rev = "5da27d32a1f522e80a3129c61f939b1934a0824a";
+    hash = "sha256-KA+Q4sk+riaFTybRQ3aO5lgPg4ECZE6G+By+x2uP/VM=";
+  };
+
+  patchPhase = ''
+    substituteInPlace ./setup.cfg --replace-fail "protobuf==3.20.1" "protobuf"
+  '';
+
+  pyproject = true;
+
+  nativeBuildInputs = [ python3Packages.setuptools ];
+
+  propagatedBuildInputs =
+    let
+      protobuf_4_21_12 = python3Packages.protobuf.overrideAttrs (_: _: { version = "4.21.12"; });
+      protobufPy = protobuf_4_21_12.override { protobuf = protobuf_21; };
+    in
+    with python3Packages;
+    [
+      ffmpy
+      music-tag
+      pillow
+      tabulate
+      tqdm
+      (librespot.override { protobuf = protobufPy; })
+      pwinput
+      protobufPy
+    ];
+
+  doCheck = false;
+
+  meta = {
+    description = "A fast and customizable music and podcast downloader.";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes
```suggestion
    description = "Fast and customizable music and podcast downloader";
```